addons/website_slides/controllers/main.py

```diff
@@ -508,6 +508,10 @@ def slide_channel_join(self, channel_id):
             return {'error': 'join_done'}
         return success
 
+    @http.route(['/slides/channel/leave'], type='json', auth='public', website=True)
```

Comment:
Normally auth user should be sufficient as you have to be logged to leave / join / subscribe / unsubscribe a channel.

Same for other routes.